project/plugins.sbt

```diff
@@ -2,7 +2,10 @@ resolvers ++= DefaultOptions.resolvers(snapshot = true)
 
 resolvers += Resolver.typesafeRepo("releases")
 
+resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Any reason not to use `resolvers += Resolver.sonatypeRepo("snapshots")`?

Reply:
Done. :-)